_docs-2-0/administration/caching.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Caching
+category: administration
+order: 11
+---
+
+Accumulo tablet servers have a **block cache** that buffers data in memory to limit reads from disk.
+This caching has the following benefits:
+
+* reduces latency when reading data
+* helps alleviate hotspots in tables
+
+The block cache stores index and data blocks. A typical Accumulo read will perfrom a binary search
```

Comment:
What is "a typical Accumulo"?

Reply:
Never mind. I got confused with the phrasing. Perhaps "A typical read operation" instead?

Reply:
Fixed in 4c22f3edc2b6